JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeRoute.kt

```diff
@@ -210,3 +204,40 @@ private enum class HomeScreenType {
     Feed,
     ArticleDetails
 }
+
+@Composable
```

Comment:
Nit: This doesn't need to be `@Composable`, right?

Reply:
Good catch! It used to be but not after I refactored the code thanks